huggingface/pytorch/buildspec-neuron.yml

```diff
@@ -0,0 +1,41 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+region: &REGION <set-$REGION-in-environment>
+base_framework: &BASE_FRAMEWORK pytorch
+framework: &FRAMEWORK !join [ "huggingface_", *BASE_FRAMEWORK]
+version: &VERSION 1.9.1
+short_version: &SHORT_VERSION 1.9
+
+repository_info:
+  inference_repository: &INFERENCE_REPOSITORY
+    image_type: &INFERENCE_IMAGE_TYPE inference
+    root: !join [ "huggingface/", *BASE_FRAMEWORK, "/", *INFERENCE_IMAGE_TYPE ]
+    repository_name: &REPOSITORY_NAME !join ["pr", "-", "huggingface", "-", *BASE_FRAMEWORK, "-", *INFERENCE_IMAGE_TYPE, "-", neuron]
+    repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/, *REPOSITORY_NAME ]
+
+context:
+  inference_context: &INFERENCE_CONTEXT
+    neuron-entrypoint:
+      source: ../../build_artifacts/inference/neuron-entrypoint.py
+      target: neuron-entrypoint.py
+    config:
+      source: ../../build_artifacts/inference/config.properties
+      target: config.properties
+    neuron-monitor:
+      source: ../../build_artifacts/inference/neuron-monitor.sh
+      target: neuron-monitor.sh
+
+images:
+  BuildHuggingFacePytorchCpuPy38InferenceDockerImage:
```

Comment:
Please remove Cpu reference in this file and add Neuron if applicable.

Reply:
Thanks for noticing! changed it to `BuildNeuronHuggingFacePytorchPy38InferenceDockerImage`